docs/conf.py

```diff
@@ -117,56 +117,56 @@
 """.format(sunpy)
 
 # -- Project information ------------------------------------------------------
-
 project = u'SunPy'
 author = u'The SunPy Community'
 copyright = u'{}, {}'.format(datetime.datetime.now().year, author)
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-
 # The short X.Y version.
 version = versionmod.version.split('-', 1)[0]
 # The full version, including alpha/beta/rc tags.
 release = versionmod.version
 
 try:
     from sunpy_sphinx_theme.conf import *
-
     html_sidebars = {'**': ['docsidebar.html']}
-
 except ImportError:
     html_theme = 'default'
 
-# Custom sidebar templates, maps document names to template names.
-# html_sidebars = {}
+try:
```

Comment:
why is this block of code and the `rstjinja` function not in the same place in this file?

Reply:
it could be anywhere you want

Reply:
What do you mean?